tardis/montecarlo/base.py

```diff
@@ -378,7 +382,7 @@ def run(
         # montecarlo.montecarlo_radial1d(
         #    model, plasma, self,
         #    virtual_packet_flag=no_of_virtual_packets,
-        #    nthreads=nthreads,
+        #    nthreads=self.nthreads,
```

Comment:
that is still commented out?

Reply:
@wkerzendorf this comment was already present in the code, so I didn't remove it (only changed the nthreads mention to self.nthreads)